TestProjects/UniversalGraphicsTest_Foundation/Packages/manifest.json

```diff
@@ -14,7 +14,7 @@
         "com.unity.test-framework": "1.1.18",
         "com.unity.test-framework.build": "0.0.1-preview.14",
         "com.unity.test-framework.utp-reporter": "1.0.2-preview",
-        "com.unity.testframework.graphics": "7.8.11-preview",
+        "com.unity.testframework.graphics": "https://github.cds.internal.unity3d.com/unity/com.unity.testframework.graphics.git#xr-fix-standalone-reuse-tests",
```

Comment:
I think this was for testing purposes? Please revert it to a proper semver reference before merging the PR (so that users outside of Unity can use the project).

Reply:
(same for the other manifest.json files)